From 900db66d3d03f75a412ab1184b5d7b6790c4f3af Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Tue, 2 May 2017 14:02:28 -0700
Subject: [PATCH] joyent/node-cueball#110 CHANGES.adoc was not updated for
 v2.2.6 Reviewed by: Robert Mustacchi <rm@joyent.com> Reviewed by: David
 Pacheco <dap@joyent.com> Approved by: David Pacheco <dap@joyent.com>

---
 CHANGES.adoc | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/CHANGES.adoc b/CHANGES.adoc
index 9eb3d07..5240d65 100644
--- a/CHANGES.adoc
+++ b/CHANGES.adoc
@@ -6,6 +6,17 @@ toc::[]
 
 ## v2.x
 
+### v2.2.6
+
+Maintenance release.
+
+Bugs fixed:
+
+ - #108 crash: unhandled smgr state transition -- issue which most commonly
+   happens due to HTTP requests that time out while stalled (buffers full)
+ - #109 Agent "health check ok" log msg needs domain name of pool (also adds
+   a bunch of other useful attributes to that log message)
+
 ### v2.2.5
 
 Maintenance release.
-- 
2.21.0

